modules/apps/portal/portal-inactive-request-handler/src/main/java/com/liferay/portal/inactive/request/handler/internal/InactiveRequestHandlerImpl.java

```diff
@@ -75,14 +75,11 @@ public void processInactiveRequest(
 
 		PrintWriter printWriter = httpServletResponse.getWriter();
 
-		Locale locale = _portal.getLocale(httpServletRequest);
+		String message = _language.get(
+			_portal.getLocale(httpServletRequest), messageKey,
+			StringPool.BLANK);
 
-		String message = null;
-
-		if (LanguageUtil.isValidLanguageKey(locale, messageKey)) {
-			message = LanguageUtil.get(locale, messageKey);
-		}
-		else {
+		if (Validator.isNull(message)) {
```

Comment:
@julschong Here, it is not a simple replacing LanguageUtil with _language,it is kind of risky, suppose you have have something wrong with this commit, you have to figure out the reason in changes for 60 files.
Please do simple replacement in this commit and you and add another commit to make other changes, in this case, this kind of change is highlighted.

Tina.

Reply:
@tinatian 
will separate this change in a second commit.